Snakefile.gsc.smk

```diff
@@ -1,12 +1,23 @@
 import os
 import sys
+from types import SimpleNamespace
 
-include: os.path.join(workflow.basedir, "Snakefile")
+sys.path.insert(0, workflow.basedir)
+scripts_dir = os.path.join(workflow.basedir, "scripts")
+sys.path.append(scripts_dir)
+from constants import VERSION as pipeline_ver
+from gsc_build_config import CONFIG_BASENAME, get_biopsy_dna_tumour_normal, get_gsc_output_folder
+from gsc_build_config import main as build_config
+
+# start with defaults
+configfile: os.path.join(workflow.basedir, "CONFIG.txt")
+module ploid:
```

Comment:
It seems like commenting out lines 14-16 doesn't interfere with a dry-run of POG965 biop2 or POG010 biop1. Why do we define the module twice in the workflow?

Reply:
No reason anymore - I forgot to delete those lines.  Good catch.